arch/M68K/M68KDisassembler.c

```diff
@@ -2789,7 +2792,8 @@ static void d68020_mull(m68k_info *info)
 
 	op1->address_mode = M68K_AM_NONE;
 	op1->type = M68K_OP_REG_PAIR;
-	op1->register_bits = (reg_0 << 4) | reg_1;
+	op1->reg_pair.reg_0 = reg_0;
+	op1->reg_pair.reg_1 = reg_1;
```

Comment:
Make sure to update this code also https://github.com/ibabushkin/capstone/blob/next/arch/M68K/M68KInstPrinter.c#L114 after these changes.